InternetHospital.BusinessLogic/Services/FilesService.cs

```diff
@@ -16,17 +16,22 @@ public class FilesService : IFilesService
     {
         private readonly IHostingEnvironment _env;
         private readonly UserManager<User> _userManager;
+        private readonly ApplicationContext _context;
 
         const int MIN_HEIGHT = 150;
         const int MAX_HEIGHT = 3000;
         const int MIN_WIDTH = 150;
         const int MAX_WIDTH = 3000;
         const int IMAGE_MAX_LENGTH = 20;
+        const string PASSPORT = "Passport";
+        const string DIPLOMA = "Diploma";
+        const string LICENSE = "License";
 
-        public FilesService(IHostingEnvironment env, UserManager<User> userManager)
+        public FilesService(IHostingEnvironment env, UserManager<User> userManager, ApplicationContext context)
         {
             _env = env;
```

Comment:
_env isn't used

Reply:
be attentive, please. this file is being used

Reply:
sorry my fault